magpie-check-inputs

```diff
@@ -1185,6 +1188,38 @@ then
     __Magpie_check_dependencies_not_setuponly "Hadoop" "Tachyon"
 fi
 
+
+#
+# Check Hive Inputs
+#
+
+if [ "${HIVE_SETUP}" == "yes" ]
+then
+    __Magpie_check_must_be_set "JAVA_HOME" "for Hive"
```

Comment:
I suspect there are more input checks that are necessary here.  For ant and stuff?  But we can deal with that later.